### DISCUSSION TITLE
User Presets v1.6.3

### DISCUSSION THREAD

**Discussion by mmunchinski** [OrcaSlicer/OrcaSlicer#1166] [Q&A]
Did something change regarding preset handling and v1.6.3?  For some reason my user defined presets for Process did not carry over but the Filament and Printer stuff did.  Am I missing something?

**Comment by mmunchinski** [comment]
I think I figured it out, v1.6.3 must have changed the name of the default FLSun v400 Process profile.  If I match the name as it displays in v1.6.3 it works.

**Reply by fr3ak2402** [reply]
Yes, I have started to rebuild the preset. The FLSun was already one of these presets. In V 1.6.4, the final preset will be used for all printers, as I still have a lot of work to do with it,

Exactly, I have only renamed the name of the files from FLSun. if you adjust these again, then everything works again.

---------------------------------------------------
As of version V 1.6.4, all files of all printers preset will follow a uniform naming system. They will also all have a buildplate model and texture. Also a honend model.

Most importantly, they will all have more than just the 0.4 nozzle, so that other presets with different diameters will be directly available.
